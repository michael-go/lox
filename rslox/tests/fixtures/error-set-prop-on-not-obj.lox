var num = 7;
num.field = "foo";