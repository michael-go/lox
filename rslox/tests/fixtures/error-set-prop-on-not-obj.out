# exit code: 65
# stdout:

# stderr:
Runtime error: Only instances have fields.
[line 2] in <script>

